client-ts/Microsoft.AspNetCore.SignalR.Client.TS/Transports.ts

```diff
@@ -247,12 +247,12 @@ export class LongPollingTransport implements ITransport {
         }
 
         this.pollXhr = pollXhr;
-        this.pollXhr.open("GET", url, true);
+
+        this.pollXhr.open("GET", `${url}&_=${Date.now()}`, true);
```

Comment:
Is there ever a case where someone would use the LongPolling class directly instead of from HttpConnection? Then we wouldn't be guaranteed that a query part already exists

Reply:
Each poll request needs a connection id which guarantees that there is query part.